### ISSUE TITLE
Add configuration bridge to use otel with the elastic environment variables

### ISSUE THREAD

**Issue by dmathieu** [elastic/apm-agent-go#1619]
**Is your feature request related to a problem? Please describe.**

Open-source packages currently using this agent  recommend customers to use environment variables to configure their instrumentation.
See https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html

We should allow those packages to migrate to otel with no breaking changes. So we should be able to configure otel with the elastic environment variables.

**Describe the solution you'd like**

A configuration tool, either part of `apmotel`, or within a new package that would read the elastic documented environment variables, and configure OpenTelemetry with those.

Note: there will still be a change there, as OpenTelemetry doesn't support the Elastic APM protocol, only OTLP.

**Comment by jsoriano** [comment]
> Note: there will still be a change there, as OpenTelemetry doesn't support the Elastic APM protocol, only OTLP.

Good point. Given this, would still be worth to support migrating with the Elastic environment variables?

**Comment by dmathieu** [comment]
I think it still may make sense. `apm-server` does support OTLP. So as long as we document that services can only communicate with recent apm-servers (7.14 and higher it seems), switching to OTLP should be transparent.
